src/Evolution/Systems/RelativisticEuler/Valencia/Equations.cpp

```diff
@@ -0,0 +1,114 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/RelativisticEuler/Valencia/Equations.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace {
+template <size_t Dim>
+tnsr::II<DataVector, Dim, Frame::Inertial> densitized_stress(
+    const tnsr::I<DataVector, Dim, Frame::Inertial>& tilde_s,
```

Comment:
A different `tilde_s` is used in `compute_dt_u`, should they be named differently? Maybe `tilde_s_i` and `tilde_s_I`